src/qt/splashscreen.cpp

```diff
@@ -12,9 +12,9 @@
 #include <interfaces/handler.h>
 #include <interfaces/node.h>
 #include <interfaces/wallet.h>
+#include <node/ui_interface.h>
```

Comment:
It seems `qt/splashscreen.cpp` does not use any symbols from `node/ui_interface.h` directly.

Reply:
thx, removed